package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
```

Comment:
this is good for now to get around a weird npm issue, but i opened an issue upstream so we won't need to explicitly install eslint (since lab installs it for us already)

https://github.com/geek/eslint-no-shadow/issues/1